kuma/static/styles/components/payments/banner.scss

```diff
@@ -50,6 +50,11 @@
             height: $payment-banner-popover-collapsed-height;
         }
 
+        .column-container > [class^='column-'],
+        .column-container > [class^='column-']:last-child {
```

Comment:
I can't see why it should be necessary to explicitly override the last-child here. So consider adding a comment to clarify.

Given that we've now turned the payment banners off, this may be hard to test, and I suppose you could just omit these changes if that would be simpler.